l10n/en/products/vpn/shared.ftl

```diff
@@ -0,0 +1,75 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/
+
+vpn-shared-product-name = { -brand-name-mozilla-vpn }
+vpn-shared-subscribe-link = Get { -brand-name-mozilla-vpn }
+vpn-shared-waitlist-link = Join the Waitlist
+vpn-shared-sign-in-link = Already a subscriber?
+vpn-shared-available-countries = We currently offer { -brand-name-mozilla-vpn } in the US, the UK, Canada, New Zealand, Singapore and Malaysia.
+vpn-shared-available-countries-v2 = We currently offer { -brand-name-mozilla-vpn } in the US, the UK, Germany, France, Canada, Malaysia, New Zealand, and Singapore.
+vpn-shared-availablity-long = The VPN is currently available for { -brand-name-windows } 10, { -brand-name-android } and { -brand-name-ios } devices in the US, the UK, Canada, New Zealand, Singapore and Malaysia.
+vpn-shared-money-back-guarantee = 30-day money-back guarantee
+vpn-shared-featured-in = Featured in
```

Comment:
What is followed after this? I don't see a lot of the strings in the shared file.

Reply:
It will be a lockup of press logos (not in the page just yet until we get approvals). I'll add a comment here to provide context.